playbooks/wazuh-manager.yml

```diff
@@ -1,5 +1,9 @@
 ---
 - hosts: <WAZUH_MANAGER_HOST>
+- vars:
```

Comment:
We should add these vars to the `/defaults/main.yml` instead of declaring them in the playbook. It's not susceptible to change.

Reply:
Added in 376628a914e93cebaabf105f3208d0e57923ef03